server.cpp

```diff
@@ -53,7 +55,62 @@ void write_log(const std::string& outfile, const std::string& log_message)
     }
 }
 
-void write_response(tcp::socket& socket, const Request& request, const std::string& date)
+std::string make_message(DIR *dir, const std::string& path, const std::string& date)
+{
+        std::string file_name;
+        std::string line;
```

Comment:
Since the `line` variable really contains more than one line consider renaming it to `lines`.